src/libs/ReportActionsUtils.ts

```diff
@@ -1054,6 +1063,27 @@ function getOneTransactionThreadReportID(reportID: string, reportActions: OnyxEn
     return singleAction.childReportID;
 }
 
+/**
+ * Returns true if all transactions on the report have the same ownerID
+ */
+function hasSameActorForAllTransactions(reportID: string, reportActions: OnyxEntry<ReportActions> | ReportAction[], isOffline: boolean | undefined = undefined): boolean {
```

Comment:
Okay I'm going to be annoying here but trying to be future proof - for all functions that are going to affect avatars, can we first check if there is a `delegateAccountID` in the action, and use that instead of `actorAccountID` if it exists? We want to show the avatar of the copilot not the person they were copiloted into.

Reply:
Although cc @Expensify/design @dylanexpensify since I'm not actually sure if that should apply to report previews too?